src/main.rs

```diff
@@ -340,10 +349,10 @@ async fn main() -> anyhow::Result<()> {
     } else {
         vec![]
     };
-    let system_contracts_options = if opt.dev_use_local_contracts {
-        system_contracts::Options::Local
-    } else {
-        system_contracts::Options::BuiltIn
+    let system_contracts_options = match opt.dev_system_contracts {
+        DevSystemContracts::BuiltIn => system_contracts::Options::BuiltIn,
+        DevSystemContracts::BuiltInNoVerify => system_contracts::Options::BuiltInWithoutSecurity,
```

Comment:
Let's add some Verification Evidence of this working as expected. Perhaps the same transaction failing with `--dev-system-contracts=built-in` but succeeding with `--dev-system-contracts=built-in-no-verify`?